main.js

```diff
@@ -10,38 +10,45 @@
 // fields if submit is performed
 
-var activityTask = document.getElementById('activity-task');
-var activityMinutes = document.getElementById('activity-minutes');
-var activitySeconds = document.getElementById('activity-seconds');
-var startActivityButton = document.querySelector('.start-activity');
-var startStopButton = document.querySelector('.start-stop-button');
+var currentActivity;
+
 var allCategoryButtons = document.querySelector('.category-buttons');
 var studyButton = document.querySelector('.icon-study');
 var exerciseButton = document.querySelector('.icon-exercise');
 var meditateButton = document.querySelector('.icon-meditate');
+var activityTask = document.querySelector('.activity-task');
+var activityMinutes = document.querySelector('.activity-minutes');
+var activitySeconds = document.querySelector('.activity-seconds');
+var startActivityButton = document.querySelector('.start-activity');
+var startStopButton = document.querySelector('.start-stop-button');
+
 // ======================================
 
 
 // event listener for buttons (change color of font and change image)
 allCategoryButtons.addEventListener('click', function (event) {
-
-  if (event.target.className === 'icon-study' || event.target.className === 'icon') {
-    document.querySelector('.icon-study').innerHTML = `
-      <img class='icon' src='./assets/study-active.svg'>Study
-    `
-  } else if (event.target.className === 'icon-exercise' || event.target.className === 'icon') {
-    document.querySelector('.icon-exercise').innerHTML = `
-      <img class='icon' src='./assets/exercise-active.svg'>Exercise
-    `
-  } else if (event.target.className === 'icon-meditate' || event.target.className === 'icon') {
-    document.querySelector('.icon-meditate').innerHTML = `
-      <img class='icon' src='./assets/meditate-active.svg'>Meditate
-    `
+  if (event.target.className === 'icon-study' || event.target.className === 'icon study') {
+    document.querySelector('img.study').classList.toggle('hidden')
+    document.querySelector('img.study-active').classList.toggle('hidden')
+    currentActivity = 'Study'
+  } else if (event.target.className === 'icon-exercise' || event.target.className === 'icon exercise') {
+    document.querySelector('img.exercise').classList.toggle('hidden')
+    document.querySelector('img.exercise-active').classList.toggle('hidden')
+    currentActivity = 'Excercise';
+  } else if (event.target.className === 'icon-meditate' || event.target.className === 'icon meditate') {
+    document.querySelector('img.meditate').classList.toggle('hidden')
+    document.querySelector('img.meditate-active').classList.toggle('hidden')
+    currentActivity = 'Meditate'
   }
 })
 
+startActivityButton.addEventListener('click', startActivityFunc)
 
+function startActivityFunc() {
+  new Activity(currentActivity, activityTask.value, activityMinutes.value, activitySeconds.value, false);
+}
```

Comment:
Looking at this..Im wondering if we need to have this new instance be inside of a function because its only one line.  Thoughts?